### ISSUE TITLE
no-import-module-exports triggered by function parameter name module

### ISSUE THREAD

**Issue by jlchereau** [import-js/eslint-plugin-import#2296]
no-import-module-exports rule is triggered by:

```
.then(module)=> {
  $.extend(true, i18n[locale], module.default);
});
```

There is a workaround, which is to rename module, but the issue is not obvious to identify and fix.

See https://github.com/import-js/eslint-plugin-import/issues/760
